### DISCUSSION TITLE
Redash started to show unrecognized error

### DISCUSSION THREAD

**Discussion by Va1RuS** [getredash/redash#5989] [Q&A]
Hello

I and my team have been using Redash for a few months and just today it started to show the following error:
`Error running query: Expecting property name enclosed in double quotes: line 7692 column 1 (char 341491)`
Could you please help me with this?
P.S. Even an empty query shows this, so it is not a matter of query itself

**Comment by truth4oll** [comment]
Hi @dtarg16, did you manage to find a solution?
